src/test/java/com/google/sps/servlets/LiveCourseDataTest.java

```diff
@@ -84,21 +85,38 @@ public void FindingExisitngTermEntity() {
   }
 
   @Test
-  public void GettingRatingData_Hours_Difficulty() {
+  public void GettingRatingData_AllProperties() {
     DatastoreService db = DatastoreServiceFactory.getDatastoreService();
-    Entity parent = addTermEntity(db);
-    addRatingEntity(db, /* hours */ 12, /* difficulty */ 7, parent.getKey());
-    addRatingEntity(db, /* hours */ 7, /* difficulty */ 4, parent.getKey());
-
-    List<Long> expectedHoursList = new ArrayList(Arrays.asList((long) 12, (long) 7));
-    List<Long> expectedDifficultyList = new ArrayList(Arrays.asList((long) 7, (long) 4));
-
-    List<Long> actualHoursList = liveCourseData.getDataFromTermRating(db, parent, "hours");
-    List<Long> actualDifficultyList =
-        liveCourseData.getDataFromTermRating(db, parent, "difficulty");
-
-    assertEquals(expectedHoursList, actualHoursList);
-    assertEquals(expectedDifficultyList, actualDifficultyList);
+    createRequest(
+        request, /*schoolName*/
```

Comment:
Nit: Comments should be in the form `/* ... */`, that way prettier will put them on the right line